include/mapnik/marker.hpp

```diff
@@ -45,8 +46,7 @@ namespace svg { struct path_attributes; }
 
 using svg::svg_path_adapter;
 
-using svg_attribute_type = agg::pod_bvector<svg::path_attributes>;
-using svg_attribute_ptr = std::shared_ptr<agg::pod_bvector<svg::path_attributes>>;
+using svg_attribute_type = std::vector<svg::path_attributes>;
```

Comment:
what is the original intent of `agg::pod_bvector` type?

Reply:
It's a class declared in agg to 'store Plain Old Data'. TBH, I only see issues with it when comparing it with std::vector. My bet is that when the library was built they decided not to use the containers from the standard library; there was a time when the performance of it was considered pretty bad, but that's no longer true.